ghub-graphql.el

```diff
@@ -174,12 +175,14 @@ data as the only argument."
                         :username username
                         :auth     auth
                         :host     host
+                        :headers  headers
                         :forge    forge
```

Comment:
I feel that `host` and `forge` should remain to each other, here and elsewhere, because only taken together the fully specify the other side we are communicating with.

Reply:
yes sure I'll do the change in the next version.